domains/mahdiruiz.domain.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "My github pages domain.",
+    "domain": "is-cool.dev",
+    "subdomain": "mahdiruiz",
+
+    "owner": {
+        "repo": "https://github.com/username/repo",
+        "email": "mahdiruiz@duck.com"
+    },
+
+    "record": {
+        "TXT": ["hostname: _github-pages-challenge-Mrdev88 Value: 0f7c7ef844983a6375d82db2f578ac"]
```

Comment:
TXT records alone doesn't do anything.

Reply:
where do i add the hostname

Reply:
That should be a separate file with that name.